src/Font Dialog/main.vala

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4
+
+private Gtk.FontDialog dialog_custom;
+
+async void on_clicked () {
+    try {
+        var family = yield dialog_custom.choose_family (workbench.window, null, null);
```

Comment:
Use `Pango.Family` here instead of var, as the type is not clear with the method call